Bottom: 2223e2666f8e18ac0b1f57de604f1f0acd56dd1c
Top:    48fc2dbd8f63b02af7633d83551dcc51f642e3e8
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-13 18:19:40 +0000

Refresh of temp-polygon-mess

---

diff --git a/src/polygon1.c b/src/polygon1.c
index c10e622..9419e57 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,7 +39,7 @@
       are marked
 */
 
-#undef NDEBUG
+//#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
